attributes/default.rb

```diff
@@ -84,7 +84,7 @@
 default['cfncluster']['armpl']['platform'] = value_for_platform(
   'centos' => { '~>8' => 'RHEL-8' },
   'amazon' => { '2' => 'RHEL-8' },
-  'ubuntu' => { '18.04' => 'Ubuntu-16.04' }
+  'ubuntu' => { '>=18.04' => 'Ubuntu-16.04' }
```

Comment:
not related to this PR but why we are using `Ubuntu-16.04` archive for both 18.04 and 20.04?

Reply:
It was the only available Ubuntu platform. Now I see that there is a new version, released on Mar 30, which should target Ubuntu 18.04. We might want to do the upgrade as a separate patch.